PacketTunnelProvider/en.lproj/Localizable.strings

```diff
@@ -10,3 +10,6 @@
 /* The body of the notification shown when Network Protection reconnects successfully */
 "network.protection.success.notification.body" = "Network Protection is On. Your location and online activity are protected.";
 
+/* The body of the notification shown when Network Protection connects successfully with the city + state/country as formatted parameter */
+"network.protection.success.notification.subtitle.including.serverLocation" = "Routing device traffic through \(serverLocation).";
```

Comment:
@graeme I noticed this appearing after building, presumably as the user text build phase step runs - committing the change here even though it isn't directly related to this PR.